File: Part 2/B. New Year's Number.cpp
//Solver : Nahin the Coder
//Date   : 02-08-2021
#include <bits/stdc++.h>
using namespace std;

int main() {
    
    int test;
    cin>>test;
    while(test--)
    {
        int n;
        cin>>n;
        if(n/2020 >= n%2020)
        {
            cout<<"YeS"<<endl;
        }
        else
        {
            cout<<"NO"<<endl;
        }
    }
    return 0;
}

File: Solved by Python/B. New Year's Number.py
# Nahin the Coder
# Date : 02 - 08 - 2021


testcase = int(input())
while testcase > 0:
    n = int(input())
    if n // 2020 >= n % 2020:
        print("YES")
    else:
        print("NO")
    testcase = testcase - 1
